### ISSUE TITLE
unable to use voice control in the list of notifications in the home screen

### ISSUE THREAD

**Issue by cvarjao** [bcgov/bc-wallet-mobile#854]
Missing for notifications in general. 
I can't use voice control or voiceover to open (view button) or dismiss ("x" button) notifications

**Comment by wadeking98** [comment]
Resolved by: https://github.com/hyperledger/aries-mobile-agent-react-native/pull/795
